test/unit/merge.js

```diff
@@ -2182,11 +2206,13 @@ describe('Merge', function() {
           .metadir()
           .path('SRC_DIR')
           .upToDate()
+          .remoteId(dbBuilders.id())
           .create()
         const child = await builders
           .metadata()
           .path(`SRC_DIR/${childName}`)
           .upToDate()
+          .remoteId(dbBuilders.id())
```

Comment:
Shouldn't `.upToDate()` imply `.remoteId()`?

Reply:
I've thought about it yes. I wanted to avoid creating new errors in other tests but that would be desirable.